Tests/UnitTests/Core/Utilities/HelpersTests.cpp

```diff
@@ -13,6 +13,8 @@
 
 #include <bitset>
 
+#include <boost/mpl/list.hpp>
```

Comment:
For a test it's probably fine, but we phased out usage of boost mpl at some point.

Reply:
Ah, sorry! I can change it to `std::tuple` instead, which is also supported by Boost test. :smile:

Reply:
Resolved.